kinto/core/storage/postgresql/__init__.py

```diff
@@ -654,6 +655,9 @@ def _format_conditions(self, filters, id_field, modified_field,
                 if len(value) == 0:
                     value = (None,)
 
+            if filtr.operator == COMPARISON.LIKE:
+                value = '%{0}%'.format(value)
```

Comment:
We need to add tests for the storage as well so that we can test that storage backends correctly supports the new operator. See: https://github.com/Kinto/kinto/blob/master/tests/core/test_storage.py#L386